apps/api/v2/tsconfig.json

```diff
@@ -18,6 +18,7 @@
     "strict": true,
     "noImplicitAny": true,
     "forceConsistentCasingInFileNames": false,
-    "noFallthroughCasesInSwitch": false
+    "noFallthroughCasesInSwitch": false,
+    "allowJs": true
```

Comment:
Because we are using platform constants package which is compiled into .js, and we are importing the .js files in the end. Jest was giving a warning about it.

Reply:
Could ignore but then would always have that warning during tests.

Reply:
let's remove allowJs for now please, I'd rather find a more local solution than impact the tsconfig of the whole project with this flag

Reply:
Agreed. Instead, I made "ts-jest" to ignore .js files in [ignore js files in ts-jest](https://github.com/calcom/cal.com/pull/12814/commits/b759275a9c86745e659a3629698ef2750628cf07)